### PR TITLE
add chinese doc

### PR DESCRIPTION
**Pull request by dengmingtong**
Issue #, if available:

Description of changes:

[rendered version](https://dengmingtong.github.io/service-workbench-on-aws-cn/installation_guide/overview)

Checklist:

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you successfully deployed to an AWS China account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] If new dependencies have been added, have they been pinned to specific versions?
- [ ] Is this change also required on the AWS Solution version?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?
- [ ] If you are updating the changelog and vending out a new release, have you updated versionNumber and versionDate in [.defaults.yml](../main/config/settings/.defaults.yml)

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/docs/zh/best_practices/cicd.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+按照以下步骤部署 CI/CD 管道:
+
+### 1. 将 `cicd-source` 堆栈部署到源帐户
+
+首先在 `cicd/cicd-source/config/settings` 中为您要为其创建 CI/CD 管道的环境创建一个设置文件。例如，要为 `dev` 环境创建 CI/CD 管道，请在 `cicd/cicd-source/config/settings/` 中创建一个 `dev.yml` 文件。
+
+您可以通过复制示例 `demo.yml` 文件来创建设置文件。请根据您的环境调整设置。阅读文件中的内联注释以获取有关每个设置的信息。
+在刚刚创建的设置文件中，将以下设置设置为`"*"`
+
+```重击
```
Comment (zxkane) [inline docs/docs/zh/best_practices/cicd.md]: 重击？这里通常是脚本名字，不需要翻译

File: docs/docs/zh/best_practices/cicd.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+工件S3BucketArn:“*”
```
Comment (zxkane) [inline docs/docs/zh/best_practices/cicd.md]: 配置代码需要翻译？

File: docs/docs/zh/best_practices/cicd.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+id: cicd
```
Comment (zxkane) [inline docs/docs/zh/best_practices/cicd.md]: 拿不准的先不翻译了？

File: docs/docs/zh/best_practices/cicd.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+您现在可以部署 `cicd-source` 堆栈。
+
+```重击
+cd cicd/cicd-源
+pnpx sls deploy --stage <环境名称>
+```
+
+### 2. 将 `cicd-pipeline` 堆栈部署到目标帐户
+
+在 ` 中创建一个设置文件
```
Comment (zxkane) [inline docs/docs/zh/best_practices/cicd.md]: 没有完整翻译？

File: docs/docs/zh/development/introduction.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+id: introduction
+title: Local Development Introduction
+sidebar_label: Introduction
+---
+
+部署应用程序和 UI 后，您可以开始在计算机上进行本地开发。 您将运行使用相同 lambda 函数代码的本地服务器。 要开始本地开发，请运行以下命令来运行本地服务器:
+
+```重击
```
Comment (zxkane) [inline docs/docs/zh/development/introduction.md]: 代码部分不用翻译

File: docs/docs/zh/development/introduction.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+$ pnpx sls 离线 -s $STAGE
+$ 光盘 -
+```
+
+然后，在单独的终端中，运行以下命令来启动 ui 服务器并打开浏览器:
+
+```重击
+$ cd 解决方案/用户界面
+$ pnpx sls start-ui -s $STAGE
+$ 光盘 -
```
Comment (zxkane) [inline docs/docs/zh/development/introduction.md]: 中断了？

File: docs/docs/zh/installation_guide/uninstall.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+id: uninstall
+title: Uninstalling Service Workbench
+sidebar_label: Uninstalling Service Workbench
+---
+
+按照这些准则删除以下内容以卸载 Service Workbench:
+
+**CloudFormation 堆栈**
+
++ 对于正在运行的工作区，在成功删除堆栈之前手动删除 PVRE 角色添加。
```
Comment (zxkane) [inline docs/docs/zh/installation_guide/uninstall.md:11]: 为啥有 PVRE，英文也这样？

**Review by zxkane — CHANGES_REQUESTED** [review]
please also add a rendered link in PR's description

File: docs/docs/zh/best_practices/cicd.md (hunk 2 above)
Comment (dengmingtong) [reply docs/docs/zh/best_practices/cicd.md]: Removed chinese best practices doc.

File: docs/docs/zh/best_practices/cicd.md (hunk 3 above)
Comment (dengmingtong) [reply docs/docs/zh/best_practices/cicd.md]: Removed chinese best practices doc.

File: docs/docs/zh/best_practices/cicd.md (hunk 4 above)
Comment (dengmingtong) [reply docs/docs/zh/best_practices/cicd.md]: Removed chinese best practices doc.

File: docs/docs/zh/development/introduction.md (hunk 5 above)
Comment (dengmingtong) [reply docs/docs/zh/development/introduction.md]: 已经修改了

File: docs/docs/zh/development/introduction.md (hunk 6 above)
Comment (dengmingtong) [reply docs/docs/zh/development/introduction.md]: 已经修改了

File: docs/docs/zh/installation_guide/uninstall.md (hunk 7 above)
Comment (dengmingtong) [reply docs/docs/zh/installation_guide/uninstall.md:11]: 是的，英文里也有
`+ For Workspaces that are running, manually delete the PVRE role additions before the stack is successfully deleted.`

File: docs/docs/zh/best_practices/cicd.md (hunk 1 above)
Comment (dengmingtong) [reply docs/docs/zh/best_practices/cicd.md]: Removed chinese best practices doc.

File: docs/docs/zh/installation_guide/architecture.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
+---
+id: architecture
+title: Service Workbench 架构
+sidebar_label: Service Workbench 架构
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Service Workbench 集成了现有的 AWS 服务，例如 [Amazon CloudFront](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Introduction.html)、[AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html) 和 [AWS Step Functions](https://docs.aws.amazon.com/lambda/latest/dg/lambda-stepfunctions.html)。 Service Workbench 使您能够创建自己的自定义的研究环境模板并与其他组织共享这些模板。为了提供成本透明度，Service Workbench 已与 [AWS Cost Explorer](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/ce-getting-started.html)、[AWS Budgets](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/budgets-managing-costs.html) 和 [AWS Organizations](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/consolidated-billing.html)。
```
Comment (zxkane) [inline docs/docs/zh/installation_guide/architecture.md]: 缺少 `集成`

File: docs/docs/zh/installation_guide/architecture.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+### 主账号
+
+这是部署 Service Workbench 基础架构的主帐户。
+
+<img src={useBaseUrl('img/deployment/installation/main_account.png')} />
+
+### 主机账户
```
Comment (zxkane) [inline docs/docs/zh/installation_guide/architecture.md]: Hosting account -> `主机账号` 合适吗？

File: docs/docs/zh/installation_guide/architecture.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+
+这是部署计算资源的帐户。
+
+<img src={useBaseUrl('img/deployment/installation/hosting_account.png')} />
+
+### 身份认证
+
+AWS 上的 Service Workbench 可以使用 OIDC IdP 作为身份验证来源。
```
Comment (zxkane) [inline docs/docs/zh/installation_guide/architecture.md:25]: Cognito 彻底不提了吗？

File: docs/docs/zh/installation_guide/installation/pre-installation/oidc-providers.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+
+以下是在 Authing、Keycloak 和 Okta 中创建 OIDC 提供程序的示例过程，以供参考。您也可以使用其他 OIDC 提供程序。
+
+- (Authing Option) [Authing][authing] 是第三方认证提供者的一个例子。
+- (Keycloak on AWS Option) [Keycloak on AWS][keycloak-solution] 是 AWS 维护的解决方案，可以作为身份验证提供者。
+- (Okta 选项）[Okta][okta] 是一个值得信赖的平台，可以保护从客户到您的员工的所有身份。
+
+按照以下步骤创建 OIDC 客户端，并获取`client_id`, `issuer`, `user_name`, `first_name`和`last_name`
+
+### (Authing Option) Authing.cn OIDC 客户端
```
Comment (zxkane) [inline docs/docs/zh/installation_guide/installation/pre-installation/oidc-providers.md]: `在Authing.cn创建用户` 是不是应该合并进来？

Comment (zxkane) [reply docs/docs/zh/installation_guide/installation/pre-installation/oidc-providers.md]: Keycloak, Okta 下同

**Review by zxkane — CHANGES_REQUESTED** [review]
docs/ 是使用 yarn 管理的项目，不应该加入新的 `pakcage-lock.json`

File: docs/docs/user_guide/sidebar/common/studies/introduction.md [hunk 12]
```diff
@@ -14,7 +14,7 @@ There are three types of Studies available in Service Workbench. You can mount m
...
 | **Organization Studies** | Specifies Studies that can be shared with other users.  It contains data that had been collected by an organization or is licensed. You can grant or deny users access to this data in order to comply with regulations or licensing restrictions on the data. |
-| **Open Data**            | Service Workbench provides access to [Open Data on AWS](https://aws.amazon.com/opendata/) data by frequently scanning the set of AWS open datasets and adding new datasets to this category. This can include 1,000 genomes and other datasets openly available through Amazon.                                                          |
+| **Open Data**            | Service Workbench provides access to [Open Data on AWS](https:// aws.amazon.com/opendata/) data by frequently scanning the set of AWS open datasets and adding new datasets to this category. This can include 1,000 genomes and other datasets openly available through Amazon.                                                          |
```
Comment (zxkane) [inline docs/docs/user_guide/sidebar/common/studies/introduction.md]: link is broken, why not finding by the strict mode of docusaurus?

**Review by zxkane — CHANGES_REQUESTED** [review]

File: docs/docs/zh/installation_guide/architecture.md (hunk 8 above)
Comment (dengmingtong) [reply docs/docs/zh/installation_guide/architecture.md]: Added

File: docs/docs/zh/installation_guide/architecture.md (hunk 10 above)
Comment (dengmingtong) [reply docs/docs/zh/installation_guide/architecture.md:25]: 在中文版里面把Cognito删掉了。

File: docs/docs/zh/installation_guide/installation/pre-installation/oidc-providers.md (hunk 11 above)
Comment (dengmingtong) [reply docs/docs/zh/installation_guide/installation/pre-installation/oidc-providers.md]: 在 ### (Authing Option) Authing.cn OIDC 客户端 章节的后面紧接着就是 ### 在Authing.cn创建用户。
两个设定成平行章节
Keycloak, Okta 也一样

File: docs/docs/zh/installation_guide/architecture.md (hunk 9 above)
Comment (dengmingtong) [reply docs/docs/zh/installation_guide/architecture.md]: 修改成 “托管帐户”

File: docs/docs/zh/installation_guide/installation/pre-installation/oidc-providers.md (hunk 11 above)
Comment (dengmingtong) [reply docs/docs/zh/installation_guide/installation/pre-installation/oidc-providers.md]: 已经修改

File: docs/docs/user_guide/sidebar/common/studies/introduction.md (hunk 12 above)
Comment (dengmingtong) [reply docs/docs/user_guide/sidebar/common/studies/introduction.md]: It should be docusaurus version issue, I open an [issue](https://github.com/awslabs/service-workbench-on-aws-cn/issues/60) and fix it later.

**Review by zxkane — COMMENTED** [review]
rendered version still is stale

**Review by zxkane — APPROVED** [review]

**Pull request merged**